spec/package-spec.coffee

```diff
@@ -19,6 +19,11 @@ describe "Package", ->
       expect(pack.incompatibleModules[0].name).toBe 'native-module'
       expect(pack.incompatibleModules[0].path).toBe path.join(packagePath, 'node_modules', 'native-module')
 
+    it "utilizes _atomModuleCache to get native modules and skips traversing through submodules", ->
+      packagePath = atom.project.getDirectories()[0]?.resolve('packages/package-with-both-incompatible-and-compatible-native-modules')
+      pack = new Package(packagePath)
+      expect(pack.isCompatible()).toBe true
```

Comment:
So if the package is named `package-with-both-incompatible-and-compatible-native-modules` I would expect the spec to assert that `pack.isCompatible()` is `false` since the words `incompatible-native-modules` are in the name of the package.